src/presets/src/Aircraft/AircraftPreset.h

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) 2022 FlyByWire Simulations
+// SPDX-License-Identifier: GPL-3.0
+
+#pragma once
+
+#include <MSFS\Legacy\gauges.h>
```

Comment:
I would suggest to use the standard C++ path-seperators '/' instead of '\'

Reply:
Makes sense - copy & pasted that from FadecGauge.h :D